src/basic/unit-def.h

```diff
@@ -58,6 +58,13 @@ typedef enum FreezerState {
         _FREEZER_STATE_INVALID = -EINVAL,
 } FreezerState;
 
+typedef enum UnitMarker {
+        UNIT_MARKER_NEEDS_RELOAD,
+        UNIT_MARKER_NEEDS_RESTART,
+        _UNIT_MARKER_MAX,
+        _UNIT_MARKER_INVALID = -1
```

Comment:
@poettering @keszybz Isn't the new convention to use `-EINVAL` instead of `-1`?

The following elements may need to be analyzed:
```
git grep -P "\b_.*_INVALID *=" | grep -v EINVAL
```